recipes/localstack-client/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "localstack-client" %}
+{% set version = "0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7cf28213ceeb6fb724153ceeacc08bc9d7b5295c9d200b00a7c131bf067b52f4"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Could you please change it to:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```